pkgs/applications/editors/neovim/neovim-gtk.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitHub, rustPlatform
+, gtk3
+, atk
+, pango
+, cairo
+, gdk_pixbuf
+, glib
+, gobjectIntrospection
+, wrapGAppsHook
+, gnome3
+}:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  name = "neovim-gtk-${version}";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "daa84";
+    repo = "neovim-gtk";
+    rev = "30b7fc87e1e92ac2bb102fd6af6a960fc75017ac";
+    sha256 = "14yy611b6pgy3qay0g1nnwss66xckz0ljw7ixpsqbwi0imjvda27";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook ];
+
+  propagatedBuildInputs= [
+    gnome3.defaultIconTheme
+    gtk3
+    gdk_pixbuf
+    atk
+    pango
+    cairo
+    glib
+    gobjectIntrospection
+  ];
+
+  cargoSha256 = "0zpdsq96xffdcgi4b9s5gj0v0j4riqp0frqwqv8d4sczhlp2yq31";
+
+  meta = with stdenv.lib; {
+    description = "GTK ui for neovim written in rust using gtk-rs bindings. With ligatures support.";
+    homepage = https://github.com/daa84/neovim-gtk;
+    license = with licenses; [ gpl3 ];
+    platforms = platforms.all;
```

Comment:
Do you want to maintain this package?

Reply:
Not really, I don't have any expertise in Rust and it was more because I wanted to try this UI. I might change my mind if I end up using it. Thanks for the icon fix.

Reply:
Not really, I don't have any expertise in Rust and it was more because I wanted to try this UI. I might change my mind if I end up using it. Thanks for the icon fix.

Reply:
Should we really accept packages without maintainers? If it is in the tree, someone will have to maintain it or it will become outdated soon.

If user just wants to share an expression, opening an issue or pull request should be enough.

Reply:
Yes. We have too many unmaintained packages. Probably someone will pick it up eventually.

Reply:
I would say it is a problem. How often does someone pick up a package to maintain it? I would say a large fraction of packages is unmaintained, only occasionally being updated by a random passer-by discovering Repology. This leads to burn-outs of either the passers-by or the core maintainers reviewing the PRs.

Reply:
Nixos makes it kinda hard to try programs since you need to know nix to create the packages instead of just relying on the typical rust or python ecosystem. It's also difficult to ask for perfect packages from the start (like the wrapping thing, running the tests). I believe we need an overlay with a lower entry bar than nixpkgs, close to wiki-based (since the maintainer resource is scarce).

Reply:
I have not yet concrete ideas, how this could look like. But I have created the repository: https://github.com/nixos-users/NUR/tree/master

Reply:
With github you still need someone to accept the PRs. Maybe a searchengine to search across different (user) overlays could work better, kind of what luarocks.org does. There is a default "root" repository but you can expand your search with user channels. 
For instance I have a few nix packages available locally I don't mind making public but that don't meet yet nixos standards (ns-3/lkl improvements/castxml/pygccxml/mptcpanalyzer etc).

Reply:
The idea was to accept make this an automatic registration process, maybe not even through the github ui but version controlled and only keep a central list of overlays a user can subscribe.
Basically to improve discovery and provide a communication channel between consumer and producer of an overlay and also to make adding/removing overlays easier.